src/client/Webhooks.ts

```diff
@@ -1,9 +1,80 @@
-import { PaymentsWebhook } from './PaymentsWebhook'
+import { IAsaasDeleteResponse } from '@/types/AsaasTypes';
+import {
+  IAsaasWebhook,
+  IAsaasWebhookInvoice,
+  IAsaasWebhookPayment,
+  ICreateWebhookParams,
+  IListWebhooksResponse,
+  IUpdateWebhookParams,
+  IWebhookResponse,
+} from '@/types/WebhookTypes';
+import { AxiosInstance } from 'axios';
 
-export class Webhooks {
-  readonly payments: PaymentsWebhook
+export class WebhooksAPI {
+  constructor(private apiClient: AxiosInstance) {}
 
-  constructor() {
-    this.payments = new PaymentsWebhook()
+  static parsePayload(payload: { event: string }): IAsaasWebhook | null {
```

Comment:
esse método posso mover para um `utils` se for relevante, mas achei interessante deixar anexado à API de webhooks